bqskit/ir/gates/parameterized/rsu3.py

```diff
@@ -0,0 +1,172 @@
+"""This module implements the rotations due to the generatos of SU(3)."""
```

Comment:
Module-level docstrings should describe what the module exports, where the more detailed info goes with the exported objects. So, for example, this should say: This module implements the RSU3Gate class.